hw/xbox/nv2a/gl/gloffscreen_glx.c

```diff
@@ -22,31 +22,33 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
-#include <stdlib.h>
+
 #include <stdio.h>
+#include <stdlib.h>
 #include <string.h>
 
 #include "gloffscreen.h"
 #include <X11/Xlib.h>
 
 struct _GloContext {
-    GLXDrawable     glx_drawable;
-    GLXContext      glx_context;
+    GLXDrawable glx_drawable;
+    GLXContext glx_context;
 };
 
-static Display* x_display;
-
+static Display *x_display;
 
 /* Create an OpenGL context */
 GloContext *glo_context_create(void)
 {
+    static bool initialized;
```

Comment:
Isn't this broken now?

It used to be `static bool initialized = false;` to explicitly show that it has not been initialized.
The `false` is gone now which is less descriptive (it probably still means `0` / `false` on most platforms, but is it even guaranteed?)

Reply:
As we discussed via Discord, according to checkpatch.pl static/global variables should not be initialized to `0`, `false`, or `NULL` values (as they should already have those values upon program startup). I made this change to comply with the script, but I've since changed my mind and I think XQEMU can deviate from this QEMU guideline in favor of readability.